### DISCUSSION TITLE
kickUser 사용후 연결 해제되는 문제

### DISCUSSION THREAD

**Discussion by jjungsu0822** [storycraft/node-kakao#394] [Q&A]
V4 버전에서 오픈채팅에서 홍보를 감지하면 자동으로 가리고 강퇴하는 기능을 잘 사용하던 도중에 
어느순간부터 정상적으로 처리한 이후 오류도 없이 봇의 연결이 끊어집니다.

**Comment by storycraft** [comment]
https://github.com/storycraft/node-kakao/blob/488d5a129a95ae810bbd56935db2f517ba09e7ac/examples/server-switch.ts#L1-L56 참고하세요

**Comment by minmoong** [comment]
봇이 죄책감을 느끼나 봅니다..ㅋㅋ
